<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MdiCVEData</class>
 <widget class="QWidget" name="MdiCVEData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>646</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayoutMdiSubWindow">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>1260</width>
       <height>0</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-598</y>
        <width>1244</width>
        <height>1224</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutScrollAreaWidget">
       <item alignment="Qt::AlignTop">
        <widget class="QGroupBox" name="groupBoxNVDs">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>600</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>600</height>
          </size>
         </property>
         <property name="title">
          <string>NVDs</string>
         </property>
         <layout class="QGridLayout" name="gridLayoutCVEs">
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayoutNVDsTop">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayoutNVDsTopLeft">
              <item>
               <widget class="QLabel" name="labelShowNVDs">
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Show</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxShowNs">
                <property name="minimumSize">
                 <size>
                  <width>160</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>160</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelEntriesNVDs">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>entries</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacerNVDsTop">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>648</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSearchNVDs">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelSearchNVDs">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Search:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditSearchNVDs">
              <property name="minimumSize">
               <size>
                <width>160</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>160</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonClearSearchNVDs">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="horizontalLayoutCVEsBottom">
            <item>
             <widget class="QLabel" name="labelNVDsItems">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Showing 0 to 0 of 0 items</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacerCVEsBottom">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>868</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="labelNVDsPage">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Page:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxNVDsPage">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBoxNVDsFilters">
            <property name="title">
             <string>Filters</string>
            </property>
            <layout class="QGridLayout" name="gridLayoutCVEsFilters">
             <item row="2" column="1">
              <widget class="QComboBox" name="comboBoxNVDsAttackVector"/>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="labelNVDsAttackVector">
               <property name="text">
                <string>Attack Vector</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="labelNVDsMinimumCVSS">
               <property name="text">
                <string>Minimum CVSS</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QComboBox" name="comboBoxNVDsMinimumCVSS"/>
             </item>
             <item row="2" column="0">
              <widget class="QComboBox" name="comboBoxNVDsProducts"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="labelNVDsProduct">
               <property name="text">
                <string>Product</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QTableWidget" name="tableWidgetNVDs">
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustIgnored</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>100</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Summary</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>ScoreV2</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>ScoreV3</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Modified</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Attack Vector</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxProducts">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>600</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>600</height>
          </size>
         </property>
         <property name="title">
          <string>Products</string>
         </property>
         <layout class="QGridLayout" name="gridLayoutPackages">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayoutBoxProducts">
            <item row="0" column="4">
             <widget class="QLineEdit" name="lineEditSearchProducts">
              <property name="minimumSize">
               <size>
                <width>160</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>160</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pushButtonSearchProducts">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" rowspan="2">
             <spacer name="horizontalSpacerProduct">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>682</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="labelSearchProducts">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Search:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayoutProducts">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayoutShowProductsEntries">
                <item>
                 <widget class="QLabel" name="labelShowProducts">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>40</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Show</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBoxShowProducts">
                  <property name="minimumSize">
                   <size>
                    <width>160</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>160</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelEntriesProducts">
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>50</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>entries</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="0" column="5">
             <widget class="QPushButton" name="pushButtonClearSearchProducts">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayoutProductsBottom">
            <item>
             <widget class="QLabel" name="labelProductsItems">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Showing 0 to 0 of 0 items</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacerProductsTop">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>868</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="labelProductsPage">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Page:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxProductsPage">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QTableWidget" name="tableWidgetProducts">
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustIgnored</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>170</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Vendor</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Product</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Version start</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Operator Start</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Version End</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Operator End</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
